tests/test_runlib.py

```diff
@@ -278,99 +274,111 @@ def test_record_dot_check_files_hash_dict_schema(self):
       sorted(self.full_file_path_list))
 
 
+  def raiseException():
+    if os.name == "nt":
+      raise IOError("Is developer mode enabled?")
```

Comment:
Some notes on [code style](https://github.com/secure-systems-lab/code-style-guidelines) and programming style: 
- We usually use snake_case and not camelCase for function names.
- We often start small helper functions like this with an underscore.
- Please try to choose mnemonic function names. I would maybe go for something like `_raise_win_dev_mode_error`.
- Please always add a docstring, to your functions, for short functions like this one it can be a oneliner.
- I think the error message could need some additional context. If I saw this in my error log, I would like to know why this IOError asks me about developer mode. :)
- It might be helpful (also for context) to include the error message from the original exception.